GUI/QtQuick/TxInputsModel.cpp

```diff
@@ -329,7 +329,7 @@ QUTXOList* TxInputsModel::getSelection()
    QList<QUTXO*> result;
    const double amount = outsModel_ ? outsModel_->totalAmount() : 0;
    logger_->debug("[{}] total amount: {}", __func__, amount);
-   if (amount > 0) { // auto selection
+   if (static_cast<int>(std::floor(amount * BTCNumericTypes::BalanceDivider)) > selectedBalance_) { // auto selection
```

Comment:
cast to int is wrong, BTC balance is always uint64_t or use BTCNumericTypes::satoshis